src/content/docs/en/guides/environment-variables.mdx

```diff
@@ -9,7 +9,7 @@ import ReadMore from '~/components/ReadMore.astro';
 
 Astro gives you access to [Vite's built-in environment variable support](#vites-built-in-support) and includes some [default environment variables for your project](#default-environment-variables) that allow you to access configuration values for your current project (e.g. `site`, `base`), whether your project is running in development or production, and more.
 
-Astro also provides [a virtual `astro:env` module](/en/reference/configuration-reference/#env) to use and organize your environment variables with type safety. It is available for use inside the Astro context (e.g. Astro components, routes and endpoints; UI framework components; middleware), and [managed with a schema in your Astro configuration](#astroenv).
+Astro also provides [a virtual `astro:env` module](/en/reference/configuration-reference/#env) to use and organize your environment variables with type safety. It is available for use inside the Astro context (e.g. Astro components, routes and endpoints; UI framework components; middleware), and [managed with a schema in your Astro configuration](#define-your-schema).
```

Comment:
Note, I *intentionally* put the main `#astroenv` link here because we hadn't yet provided a link down to that section! The first link goes to the reference page, but I think people should be taken down to read *about* Astro env.

Open to suggestions if you'd like to change the wording, (it doesn't have to match exactly where the link takes you!) I just wanted to entice people, and let them get directly to that section. :smile:

Reply:
I feel like having the reference first is a bit odd as it could redirect people before looking at the guide

```suggestion
Astro also provides a way to [use and organize your environment variables with type safety](#type-safe-environment-variables). It is available for use inside the Astro context (e.g. Astro components, routes and endpoints, UI framework components, middleware), and managed with a schema in your Astro configuration. You can see learn more about the `astro:env` module configuration options [in the Configuration reference](/en/reference/configuration-reference/#env).
```

Reply:
Right, makes sense!